src/events/events.service.ts

```diff
@@ -457,7 +458,16 @@ export class EventsService {
       return null;
     }
 
-    const adjustedPoints = points ?? POINTS_PER_CATEGORY[type];
+    let adjustedPoints = points ?? POINTS_PER_CATEGORY[type];
```

Comment:
This is small but we actually always need to take the points as an override because it's used by our manual control panel to create events with manually set points.

1. points
2. if deposit use manual calculation
3. otherwise use POINTS_PER_CATEGORY